UserInterface.adoc

```diff
@@ -80,52 +128,368 @@ fundamental A: If it does behave that way now, I'd think it would be wise to
+
+// TODO: Is this true?
+UP/DWN::
+    If less than the middle value, it will slide down faster than up, and vice versa.
+
+SCALE::
+    Portamento proportional scale.
+
+DEPTH::
+    Portamento proportional depth.
+
+===== Scale settings
+
+// TODO: This whole part
+This is the only settings settings that is shared by each part.
+
+ENABLE::
+    Desc
+
+PER/OCT::
+    Desc
+
+REF FREQ::
+    Desc
+
+1/1 MIDI NOTE::
+    Desc
+
+CENTER::
+    Desc
+
+NAME::
+    Desc
+
+COMMENT::
+    Desc
+
+SHIFT::
+    Desc
+
+TUNINGS::
+    Desc
+
+KEY MAPPING::
+    Desc
+
+IMPORT .SCL::
+    Desc
+
+IMPORT .KBM::
+    Desc
+
+RETUNE::
+    Desc
+
+==== Browser
+
+// TODO: Preset/Patch. Use preset, it's what's in the UI.
+// TODO: If not fixed, mention how enabling/disabling a bank shows all the patches.
+// TODO: Image
+
+Allows browsing the ZynAddSubFX collection of ready-made sounds.
+It features a search bar and 4 columns, which can all be clicked to be toggled:
+
+1. *BANK*: The actual preset collection. Hovering over each one shows you where it is in the filesystem.
+2. *TYPE*: Allows selecting a specific type of preset. Useful for filtering.
+3. *TAG*: Further filtering using tags.
+4. *PRESET*: The actual list of presets.
+
+==== Mixer
+
+// TODO: Images of master and one part strip
+
+The mixer allows mixing of each part (the part mixer strip) as well as the output signal (labeled as "MASTER", the master mixer strip).
+
+.The master strip
+
+The master mixer strip features two vertical segments,
+one being the VU meter,
+and the other being the vertical global volume fader.
+This fader is actually the same one seen in the status panel!
+
+.The part strip
+
+From top to bottom, the part strip contains:
+
+1. A PART ENABLE button, which toggles the part on and off.
+2. The same VU meter and volume fader as in the master strip, only applied to the part.
+3. Part panning. 64 is the default and is also the middle-point.
+4. Active MIDI channel dropdown, identical to the one in the <<_instrument_settings, instrument settings>>.
+5. The EDIT button, which sets the current part as the one you can edit in <<_part_settings, part settings>>.
+
+==== Kit settings
+
+Kit settings
+
+==== Macro learn panel
+
+Macro learn panel
+
+==== Effects editor
+
+Effects editor
+
+==== ADDSynth
+
+ADDSynth
+
+==== SUBSynth
+
+SUBSynth
+
+==== PADSynth
+
+PADSynth
 
 === General conventions
-The Zyn-Fusion GUI tries to maintain a consistent language through it's
-controls.
+
+The Zyn-Fusion GUI tries to maintain a consistent language through it's controls.
 The ones you'll see most commonly are:
 
 * Buttons
 * Sliders
 * Knobs
 * Number fields
-* anything else?
+* Input boxes
+
+The knobs and sliders can be reset to their default position with a double-click or a middle-mouse click.
 
-The knobs and sliders can be reset to their default position with a double-click or a middle-mouse-click.
+// TODO: No way to reset the number?
```

Comment:
heh, I would have thought middle mouse would have worked here as well, but maybe that logic is missing.

Reply:
Ah, fair enough. There's only so many buttons at the end of the day.